types/react-table/index.d.ts

```diff
@@ -4,28 +4,29 @@
 //                 Pavel Sakalo <https://github.com/psakalo>,
 //                 Krzysztof Porębski <https://github.com/Havret>,
 //                 Andy S <https://github.com/andys8>,
-//                 Grzegorz Rozdzialik <https://github.com/Gelio>
-//                 Cam Pepin <https://github.com/cpepin>
+//                 Grzegorz Rozdzialik <https://github.com/Gelio>,
+//                 Cam Pepin <https://github.com/cpepin>,
+//                 Sarid Shinwari <https://github.com/ngrok-sarid>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 import * as React from 'react';
 
 export type ReactTableFunction = (value?: any) => void;
-export type AccessorFunction<D = any> = (row: D) => any;
```

Comment:
Removing the default generic argument is a breaking change. I am not sure what the policy on that is. I am fine with this change, but I am not sure if the updated typings' version will reflect the breaking change